### ISSUE TITLE
lgsvl gps data not aligned properly in autonomousStuff map

### ISSUE THREAD

**Issue by asobhy-qnx** [lgsvl/simulator#1597]
I am experiencing localization issues when using gps on LGSVL with autoware.auto.

I replaced ndt localization with gps and imu positioning. 

I am using this map asset: https://assets.lgsvlsimulator.com/ec057870762b5a967a451c93444b67d0b64e9656/environment_AutonomouStuff

LGSVL version 2020.06

First thing I am seeing is that the altitude the gps is recording is -18.9m which does not seem right. I do not believe AutonomousStuff parking lot is 18.9 meters below surface of the ocean. From https://www.freemaptools.com/elevation-finder.htm it seems that the altitude is 14.0m at gps coordinates 37.38099,-121.90861

After offsetting for the -18.9m altitude, when I place the vehicle and on the origin on both the lanelet2 map and LGSVL map the two maps are aligned perfectly, then when I start moving the vehicle in the East direction I see the car position change in the East-South direction on the lanelet2 map. When driving East the latitude should stay constant but it does change which is not correct as seen on LGSVL GPS reading

**Comment by EricBoiseLGSVL** [comment]
Current version of autonomoustuff is rotated correctly.  Thanks for the help on the offset, next release we will have it corrected to 14m.

**Comment by asobhy-qnx** [comment]
How can I make sure I have the current version of the map?

**Comment by EricBoiseLGSVL** [comment]
release 2021.2 tag.  2021.2 map will not work with 2020.06 simulator.

**Comment by asobhy-qnx** [comment]
@EricBoiseLGSVL I tested using 2021.2 and the behaviour in 2020.06 is the same as in 2021.2.

Tested on LGSVL 2020.06 & 2021.02
The problem:
When driving on the map in East or West direction Lat value for GPS increase or decrease respectively. This indicate a misalignment in the gps data on LGSVL map.

How to reproduce:

1- Manually drive the vehicle until you get an orientation on the IMU equal to [0,0,0,1], heading of zero degrees, which puts the vehicle into a heading in the East direction, as confirmed with google maps. Don't know why the IMU has it's zero heading facing east, if it's frame of reference is matching with base_link, where x is front and y is left.

2- Once the vehicle is facing east, drive the car in straight line and observe the value for the latitude on LGSVL GPS reading, you will see that the value is changing. Shouldn't the Latitude be fixed if we are driving at a zero angle East?
